docs/detectors/vcpkg.md

```diff
@@ -0,0 +1,19 @@
+# vcpkg Detection
+
+## Requirements
+
+vcpkg detection triggers off of `vcpkg.spdx.json` files found under the scan directory. You must use a version of vcpkg in your build that generates SBOM files.
```

Comment:
Mind placing the actual version and/or link VCPKG toolchain that shipped with the SBOM functionality?